crates/biome_service/src/workspace/scanner.rs

```diff
@@ -408,7 +408,11 @@ fn open_file(ctx: &ScanContext, path: &BiomePath) {
     }) {
         Ok(Ok(())) => {}
         Ok(Err(err)) => {
-            ctx.send_diagnostic(err);
+            let mut error: Error = err.into();
+            if !path.is_config() && error.severity() == Severity::Error {
+                error = error.with_severity(Severity::Warning);
+            }
+            ctx.send_diagnostic(Diagnostic::new(error));
```

Comment:
I don't understand why we need to pass `verbose` to the workspace. Can't we change the the severity of the diagnostics at end, before printing them? 

As far as I can see, we send the diagnostic anyway... so I fail to see additional parameters inside the workspace.